libs/server/Resp/AdminCommands.cs

```diff
@@ -250,8 +250,16 @@ private bool ProcessAdminCommands<TGarnetApi>(RespCommand command, ReadOnlySpan<
             {
```

Comment:
Looks like we have 2 implementations for PING for some reason (here and in BasicCommands), I don't thing this one is ever called... Can we remove it altogether?